appinventor/components-ios/src/TableArrangement.swift

```diff
@@ -162,9 +163,10 @@ fileprivate enum ConstraintUpdate {
     super.init(frame: .zero)
     _emptyConstraints = [heightAnchor.constraint(equalToConstant: 100), widthAnchor.constraint(equalToConstant: 100)]
   }
-
+  
```

Comment:
Make sure in Xcode you set it up to not save indentation on whitespace only lines, and remove the extra spaces here.

Reply:
ok sir

Reply:
I'm still seeing this whitespace here.